server/bin/pbench-unpack-tarballs.sh

```diff
@@ -161,6 +161,18 @@ function do_work() {
             continue
         fi
 
+        pbench-check-tb-age ${link}
```

Comment:
@ndokos, isn't the `pbench-check-tb-age` script used here?

Reply:
But of course!

Reply:
We might need it, but do we need it enough right now to delay this work further?  Seems like we can take this in, and add that behavior when/if that need arises.  What do you think?

Reply:
I'm fine with that. We probably should open an issue after this gets merged.

Reply:
I'll do that; leaving this conversation unresolved to help me track that.

Reply:
Filed https://github.com/distributed-system-analysis/pbench/issues/1652.